language/Spanish/strings.po

```diff
@@ -29,26 +29,32 @@ msgid ""
 msgstr ""
 "\n"
 "\n"
-"Pulsa cualquier tecla para continuar."
+"Presiona cualquier tecla para continuar"
 
 #: src/frontend/mame/ui/ui.cpp:559
 msgid ""
 "This system requires media images to be mounted for the following device(s): "
 msgstr ""
+"Este sistema requiere que se monten imágenes de medios para el/los "
+"siguiente(s) dispositivo(s): "
 
 #: src/frontend/mame/ui/ui.cpp:1254
 #, c-format
 msgid ""
 "UI controls enabled\n"
 "Use %1$s to toggle"
 msgstr ""
+"Controles de IU habilitados\n"
```

Comment:
UI and IU can be used interchangeably in Spanish, I'd leave as UI for consistency.

Reply:
UI is the correct term in Spanish, even if the two letters are swapped around.

Reply:
If you Google Interfaz de Usuario you either get IU or UI, and most often than not it's latter. 
Same for UX btw: it's Experiencia de Usuario but nobody will ever say EU anywhere for obvious reasons ...